### PR TITLE
HBSP-197: Hubspot API error when creating a language copy for a form

### PR DESCRIPTION
**Pull request by v-ladynev**
https://bt.smartling.net/browse/HBSP-197

### CODE REVIEW

File: src/integration-test/java/com/smartling/connector/hubspot/sdk/it/FormsIntegrationTest.java [hunk 1]
```diff
@@ -152,6 +152,16 @@ public void shouldCloneForm() throws Exception
         assertFormDetailIsNotEmpty(clonedForm);
     }
 
+    // @Test
+    // This is to check API
```
Comment (dlyash) [inline src/integration-test/java/com/smartling/connector/hubspot/sdk/it/FormsIntegrationTest.java]: Please, either remove the test or uncomment `@Test`

**Comment by yskorodumov** [comment]
Could you please remove me from this email list?

2017-08-16 15:29 GMT+03:00 Dmitry L <notifications@github.com>:

File: src/integration-test/java/com/smartling/connector/hubspot/sdk/it/FormsIntegrationTest.java (hunk 1 above)
Comment (v-ladynev) [inline src/integration-test/java/com/smartling/connector/hubspot/sdk/it/FormsIntegrationTest.java]: @dlyash Thank you. Done.

**Comment by dlyash** [comment]
@yskorodumov Hi, Yegor! :) See "Unwatch" button at the top

**Pull request merged**